### PR TITLE
Fix Sanity Product V2 tests blocking CI

### PR DESCRIPTION
**Pull request by 0x346e3730**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixing CI for PRs blocked by Sanity Product V2 tests
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI should be green.
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | PrestaShop SA

Issue and fix described here : https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

### CODE REVIEW

**Review by nicosomb — DISMISSED** [review]

File: .github/workflows/skip-sanity-productV2.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - src/PrestaShopBundle/Form/Admin/Sell/Product/**
+      - src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/**
+      - src/Core/Domain/Product/**
+      - src/Adapter/Product/**
+      - src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/**
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo "Not required"'
```
Comment (boubkerbribri) [inline .github/workflows/skip-sanity-productV2.yml:23]: You maybe have to add matrix to match exactly the names of the other required workflows.

**Review by Progi1984 — DISMISSED** [review]

**Review by boubkerbribri — COMMENTED** [review]
This will run the original workflow as the path is correct https://github.com/PrestaShop/PrestaShop/pull/31628/files#diff-9ceb3d0b35e87357af695e0c74ce6f8998b5db9210f35fbeb33a54b357c87382R5-R6
You can see below my comment, and you can test this by deleting the modification on the original workflow. https://github.com/PrestaShop/PrestaShop/pull/31628/files#diff-9ceb3d0b35e87357af695e0c74ce6f8998b5db9210f35fbeb33a54b357c87382R4

**Comment by 0x346e3730** [comment]
Trying the matrix on https://github.com/PrestaShop/PrestaShop/pull/31630, this PR should skip the tests

File: .github/workflows/skip-sanity-productV2.yml (hunk 1 above)
Comment (0x346e3730) [reply .github/workflows/skip-sanity-productV2.yml:23]: Nice catch 👍

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by 0x346e3730** [comment]
Skipping correctly when files does not match the paths : https://github.com/PrestaShop/PrestaShop/actions/runs/4321799948/jobs/7543451376

**Review by jolelievre — APPROVED** [review]
Thanks @0x346e3730

**Pull request merged**